tests/test_files/README.txt

```diff
@@ -96,6 +98,14 @@ OT-PAL-8-face.dcm
 zipMR.gz
   * a gzipped version of MR_small.dcm
   * used for checking that deferred read reopens as zip again (issue 103)
+  
+dcmqi_seg.dcm 
+  * Segmentation IOD (voxel-based segmentation)
+  * http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_A.51.html
+
+dcmqi_tid1500.dcm
+* structured report element from DCMQI
```

Comment:
DICOM SR document that follows TID 1500, generated by https://github.com/QIICR/dcmqi